frontend/src/pages/HomePage.jsx

```diff
@@ -62,92 +63,95 @@ class HomePage extends Component {
         className="homePage"
       >
         <AppBar
-          id = "homePage_Header"
-          position = "sticky"
+          id="homePage_Header"
+          position="sticky"
         >
           <Toolbar>
 
             {/* this is our rentr logo */}
             <img
-              id="logo" 
+              id="logo"
               src={logo}
               alt="Rentr Logo"
             />
 
             {/* this is used to add the space between the logo and sign in button */}
             <Typography
               type="title"
-              color="inherit" 
+              color="inherit"
               style={{
-                flex: 1 
+                flex: 1
               }}
             />
-            
+
             {!this.props.status
               ?
               (
                 <React.Fragment>
                   <Button
-                    className = "homePage_Header_Login"
-                    id = "homePage_Header_Login"
+                    className="homePage_Header_Login"
+                    id="homePage_Header_Login"
                     variant="contained"
-                    onClick = {()=>{
+                    onClick={() => {
                       this.props.setLogin_dialog(true)
                       this.props.setRegister_dialog(false)
                     }}
                   >
                     Log In
                   </Button>
-                
+
                   {this.handleShowLogInDialog()}
                   {this.handleShowRegisterDialog()}
                 </React.Fragment>
               )
               :
-                this.handleShowLoggedIn()
+              this.handleShowLoggedIn()
             }
           </Toolbar>
+
+          <CreateListingButton />
```

Comment:
This is the second non-linting change to this file. There's probably a better place I could've put this, but this works for now.